config/routes.rb

```diff
@@ -24,6 +24,9 @@
     end
   end
 
+  get '/auth/github'
+  get '/auth/github/callback', to: "users#update"
```

Comment:
so that's where the api response is being routed to in our app

Reply:
well this part isnt the API yet. this is just the routing to send a user to github to authorize use to use their info